translations/expansion_content.tex/de.po

```diff
@@ -218,6 +266,25 @@ msgid ""
 "    \\end{itemize}\n"
 "  "
 msgstr ""
+"      \\draw (0, 0) node[inner sep=0] {\\makebox[\\textwidth][c]{\\includegraphics[width=0.8\\textwidth]{translations/assets/de/ballista.png}}};\n"
```

Comment:
Please try making the asset names consistent with the original, instead of comping up with your own names. This file is called `war_machine.png`. It will be easier to maintain, if we know all the names.